osmcha/changeset.py

```diff
@@ -285,6 +287,35 @@ def is_whitelisted_editor(self):
                 is_whitelisted = True
         return is_whitelisted
 
+    def autovandal(self):
+
+        reason = 'Flagged by ML classifier'
+
+        def load_model():
+            return joblib.load('models/autovandal.pkl')
+
+        def changeset_to_data(changeset):
+            return [
+                len(changeset.editor),
```

Comment:
Agree @rodowi 

I can't think of a way to do this. Posted here too: https://github.com/mapbox/enforcement-droid/issues/8#issuecomment-276583986

Basically, in the current approach I handle one changeset at a time. So, when passed to the [current `to_category` function](https://github.com/mapbox/enforcement-droid/blob/feature-tokenizer/dataprep.py#L3) would not we always get back the categories as `0` since there will be only one category from the one sample?